### ISSUE TITLE
argument-list-wrapping does not allow an inline block comment

### ISSUE THREAD

**Issue by hkurokawa** [pinterest/ktlint#926]
## Expected Behavior
Expect this code not to cause any errors.

```kotlin
fun main() {
    someMethod(
        /* firstName= */ "John",
        /* lastName= */ "Doe",
        /* age= */ 30
    )
}
```

## Observed Behavior
```
/Users/hkurokawa/temp/Test.kt:3:26: Argument should be on a separate line (unless all arguments can fit a single line) (experimental:argument-list-wrapping)
/Users/hkurokawa/temp/Test.kt:4:25: Argument should be on a separate line (unless all arguments can fit a single line) (experimental:argument-list-wrapping)
/Users/hkurokawa/temp/Test.kt:5:20: Argument should be on a separate line (unless all arguments can fit a single line) (experimental:argument-list-wrapping)
```

## Steps to Reproduce

Save the above code as `test.kt` and run the below command.

```
ktlint --verbose --experimental ~/temp/test.kt
```

## Your Environment
* Version of ktlint used: 0.39.0
* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): command line
* Version of Gradle used (if applicable): N/A
* Operating System and version: macOS 10.15.6
* Link to your project (if it's a public repository): private repository

## Remarks
Sometimes I want to use the parameter name as an inline comment at the call site, so that readers don't need to look at the method declaration to understand the parameter's purpose.  Kotlin has named argument so this technique is useful only when we are calling Java methods from Kotlin code.

**Comment by hkurokawa** [comment]
Thanks for the effective work!

**Comment by spatenotte** [comment]
Looks like a similar issue is happening in Ktlint 0.49.1, using the standard rules:
![image](https://github.com/pinterest/ktlint/assets/5433694/eaf7d0ff-5b00-4686-bb66-bef351bb6a5f)

**Comment by paul-dingemans** [comment]
> Looks like a similar issue is happening in Ktlint 0.49.1, using the standard rules:

No, that is not a similar issue. As you can see in your output, the "standard:comment-wrapping" rule is not allowing a block comment to be followed by a code element. This is intended behavior. See https://pinterest.github.io/ktlint/0.49.1/rules/standard/#comment-wrapping. 
See [how do i disable a rule](https://pinterest.github.io/ktlint/0.49.1/faq/#how-do-i-enable-or-disable-a-rule) if you want to disable that specific rule.
